language/module.md

```diff
@@ -0,0 +1,92 @@
+# Puppet Module Schema
+
+This document describes allowed (optional) and required files and directories in
+a Puppet Module Release.
```

Comment:
What is the definition of a "Puppet Module Release"?

Reply:
the .tar.gz archive of a module, that you can download from the forge for example.

Reply:
That's good context for readers of this document, and I think the definition of "Puppet Module Release" should be noted here in the document.

I see there's some confusion about if this is the spec for what's in git vs. not, etc... So defining the scope of this document up front could help to reduce confusion and clarify what exactly is being defined.

Reply:
That's at the top, the `## What is a Puppet Module`. Do you think that's not sufficient?

Reply:
🤦 Ah, I somehow missed that section entirely, sorry. The "What is a Puppet Module" section looks good to me.